src/components/subscribe/mailchimp-subscribe.stories.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import { withKnobs } from "@storybook/addon-knobs";
+import { SubscribeForm } from "./subscribe-form";
+import { MailchimpSubscribe } from "./mailchimp-subscribe";
+import PropTypes from "prop-types";
+import { Input } from "../input/input";
+
+export default { title: "Newsletter form", decorators: [withKnobs] };
+
+const Wrapper = ({ children, classes }) => (
+  <div className="columns">
+    <div className={classes || "column is-one-third"}>{children}</div>
+  </div>
+);
+
+Wrapper.propTypes = {
+  children: PropTypes.node,
+  classes: PropTypes.string
+};
+
+export const mailchimpSubscribe = () => {
+  const [chimpUrl, setChimpUrl] = useState();
```

Comment:
And we can remove the check from `L32`
```suggestion
  const [chimpUrl, setChimpUrl] = useState("");
```